Check first if the unimod_db is already on the place.
Author: Pavel Tomanek (Inuits/UGent)
--- pyteomics/mass/unimod.py.orig	2026-02-16 11:42:07.859408000 +0100
+++ pyteomics/mass/unimod.py	2026-02-16 11:43:09.218943000 +0100
@@ -26,6 +26,7 @@
 #   limitations under the License.
 
 import re
+import os
 
 from lxml import etree
 try:
@@ -712,6 +713,8 @@
             XML file will be downloaded from default location.
         """
         if path is None:
+            path = os.getenv("PYTEOMICS_UNIMOD_DB")
+        if path is None:
             self.path = None
             self.session = load(_unimod_xml_download_url)
         else:
